test/Sentry.Testing/NullAttachmentContent.cs

```diff
@@ -0,0 +1,8 @@
+namespace Sentry.Testing;
+
+internal sealed class NullAttachmentContent : IAttachmentContent
```

Comment:
nitpick: we could make the `.ctor` `private`, so that we can only use the singleton `Instance`